src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    Replay: 'Replay do começo.',
+    replay_message: 'Obrigado por assistir.',
+    requestScreenShare: 'Pedir compartilhamento de tela',
+    requestedScreenShare: 'Compartilhamento de tela foi pedido',
+    Reset: 'Redefinir',
+    Rtime: 'Tempo restante',
+    rusureCquiz: '<span>Tem certeza que deseja fechar o quiz? </span>',
+    rvtu: 'Votos recebidos/ total de usuários',
+    reply: 'Responder',
+    safariBrowserIssue: 'Seu browser Safari {virtualclass1} não é suportado. Nós suportamos completamente o Chrome e o Firefox',
+    Save: 'Salvar',
+    savesession: 'Deseja salvar a sessão atual?',
+    savesessionTechSupport: "Deseja salvar a sessão atual? <br /> Uma vez baixada a sessão, atualizações não serão disponíveis ao baixar novamente a mesma sessão.",
+    screensharealready: 'A tela está sendo compartilhada.',
+    ScreenShare: 'Compartilhamento de tela',
+    screensharemsg: 'A tela acima está sendo compartilhada.',
+    screensharenotsupport: 'Compartilhamento de tela não é suportado',
+    screensharedenied: 'Usuário negou o compartilhamento de tela',
+    second: 'Segundo',
+    selfview: 'Desabilitar visualização pública',
+    SetTimer: 'Definir cronômetro',
+    sesseionkeymissing: 'A chave da sessão está faltando no Front End.', // from javascript
+    sentPackets: 'Enviar <br/><span>Pacotes</span>',
+    // 'sessionend': "Fechar sessão.",
+    SessionEnd: 'Fim da sessão',
+    sessionexpired: 'Sessão vencida',
+    sessionendmsg: 'Sessão foi fechada. Você pode fechar seu browser agora.',
+    setting: 'Configuração',
+    Searchuser: 'Buscar usuário',
+    compartilhar: 'Compartilhar',
+    ShareVideo: 'Compartilhar vídeo',
+    shareAnotherYouTubeVideo: 'Compartilhar outro vídeo do YouTube',
+    SharePresentation: 'Compartilhar apresentação',
+    SharePresentationdbHeading: 'Painel de apresentação',
+    sharetoall: 'Habilitar visualização pública',
+    Showresulttostudents: 'Mostrar resultado aos estudantes',
+    sitePoll: 'Enquete a nível de site',
+    Skip: 'Fechar',
+    someproblem: 'Há um problema no envio do arquivo. Favor tentar novamente.',
+    SourceUnavailableError: 'Favor certifique-se que a mesma Webcam não está sendo usada em múltiplos browsers ou múltiplas aplicações.',
+    // 'SourceUnavailableError' : 'Outra aplicação no seu computador pode estar usando a Webcam. Favor fechar tais aplicações.',
+    speakerTest: 'Se você consegue ouvir música, as caixas de som estão funcionando.',
+    Spoll: 'Enquete de site',
+    ssBtn: 'Parar compartilhamento',
+    ssStop: 'Parar compartilhamento de tela',
+    startnewsession: 'Você realmente quer encerrar esta sessão?',
+    status: 'Status',
+    stdPublish: 'stdPublish',
+    stdscreenshare: 'Deseja compartilhar sua tela?',
+    studentAudEnable: 'Áudio do estudante habilitado',
+    studentSafariBrowserIssue: 'Seu browser {virtualclass1} {virtualclass2} não pode compartilhar sua Webcam com outros usuários. Congrea suporta completamente Chrome e Firefox.',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 179. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)